http/src/test/scala/filodb/http/ClusterApiRouteSpec.scala

```diff
@@ -83,21 +81,9 @@ class ClusterApiRouteSpec extends FunSpec with ScalatestRouteTest with AsyncTest
 
     it("should return shard status after dataset is setup") {
       setupDataset()
-      // Repeatedly query cluster status until we know it is OK
-      var statuses: Seq[ShardStatus] = Nil
-      do {
-        probe.send(clusterProxy, NodeClusterActor.GetShardMap(dataset6.ref))
-        Thread sleep 500
-        statuses = probe.expectMsgPF(3.seconds) {
-          case CurrentShardSnapshot(_, mapper) => mapper.statuses
-        }
-        println(s"Current statuses = $statuses")
-        info(s"Current statuses = $statuses")
-        if (statuses.exists(_ == ShardStatusError)) {
-          info(s"ERROR in status, breaking")
-          throw new RuntimeException(s"Got error in statuses $statuses")
-        }
-      } while (statuses.take(2) != Seq(ShardStatusActive, ShardStatusActive))
+
+      // Give the coordinator nodes some time to get started
+      Thread sleep 1000
```

Comment:
This seems much simpler than what was done before, but is this sleep long enough? Can both strategies be used?

Reply:
The problem is that the wait loop is not really reliable, and in any case we don't really care about the exact status in this test.  We just want to check that the results are properly formed in HTTP.